Sources/Moya/Plugins/AccessTokenPlugin.swift

```diff
@@ -28,15 +37,11 @@ public struct AccessTokenPlugin: PluginType {
     /// The access token to be applied in the header.
     public let token: String
 
-    private var authVal: String {
-        return "Bearer " + token
-    }
-
     /**
      Initialize a new `AccessTokenPlugin`.
 
      - parameters:
-       - token: The token to be applied in the pattern `Authorization: Bearer <token>`
+       - token: The token to be applied in the pattern `Authorization: <AuthorizationType> <token>`
     */
     public init(token: String) {
```

Comment:
I was thinking also about this `token` here. Normally you don't really have your token when initializing the `MoyaProvider`. Could we use a closure here that will get us a token? Marking it with `autoclosure` should also allow people with token upfront to have a nicer migration. 

What do you think?